linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands/ansible/shared.yml

```diff
@@ -30,7 +30,7 @@
 - name: Overwrites the rule in rules.d
   lineinfile:
     path: "{{ item.1.path }}"
-    line: '-a always,exit -F path={{ item.0.item }} -F perm=x -F auid>={{{ auid }}} -F auid!=unset -F key=privileged'
+    line: '-a always,exit -F path={{ item.0.item }} -F perm=x -F auid>={{{ auid }}} -F auid!=unset -F key=special-config-changes'
```

Comment:
Please, try to keep the PRs self-contained. These changes on the key could be separated from changes to the `ospp,profile` file.

Reply:
And they also need to be updated in the bash remediations.

Reply:
Then I'd have to create another ticket to track that change ;)

But how do you update the bash remediations? Looks like ``shared/bash_remediation_functions/perform_audit_rules_privileged_commands_remediation.sh`` does not parse ``key`` fields.

Reply:
The function does add the key `privileged` as defined in here https://github.com/ComplianceAsCode/content/blob/71c5a326c758a0f107e4077776dc81342fc46aef/shared/bash_remediation_functions/perform_audit_rules_privileged_commands_remediation.sh#L87